examples/context-example.php

```diff
@@ -109,5 +110,131 @@ function matchContextDemo($authProvider, $projectId, $contextUid)
     return $response;
 }
 
+/**
+ * @param \Smartling\AuthApi\AuthApiInterface $authProvider
+ * @param string $projectId
+ * @param string $fileUri
+ * @return array
+ */
+function uploadAndMatchContextDemo($authProvider, $projectId, $fileUri)
+{
+    $response = FALSE;
+    $context = \Smartling\Context\ContextApi::create($authProvider, $projectId);
+    $params = new UploadContextParameters();
+    $params->setContextFileUri($fileUri);
+    $params->setName('context.html');
+    $st = microtime(true);
+
+    try {
+        $response = $context->uploadAndMatchContext($params);
+    } catch (\Smartling\Exceptions\SmartlingApiException $e) {
+        var_dump($e->getErrors());
+    }
+
+    $et = microtime(TRUE);
+    $time = $et - $st;
+
+    echo vsprintf('Request took %s seconds.%s', [round($time, 3), "\n\r"]);
```

Comment:
It looks strange to insert `\n\r` into the string. Just curious, is there some reason why it cannot be part of string?